scripts/run-github-super-linter

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+# Run [GitHub Super-Linter](https://github.com/github/super-linter/blob/master/docs/run-linter-locally.md) locally to lint Groovy code
+docker run -e RUN_LOCAL=true -e VALIDATE_GROOVY=true -e VALIDATE_BASH=true "$@" -v "$(pwd)"/:/tmp/lint github/super-linter
```

Comment:
We've been storing scripts in the `/resources/uk/gov/defra/ffc/scripts` folder so suggest moving to there.

Reply:
The reason I've placed the script in `./scripts` as opposed to the resource directory is so other repositories, which use `./scripts`, can re-use the `runGitHubSuperLinter` task without having to make additional changes.

Reply:
Is it just the volume mounting that would need changes?  If so we could do something like we do in some of our scripts and add a line to determine the root directory where the scan should run to and pass that to the docker command?

For example in https://github.com/DEFRA/ffc-demo-development/blob/master/scripts/start, the equivalent would look like the below.  Not saying that would work in this instance but could we do similar?
```
projectRoot=$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)

docker run -e RUN_LOCAL=true -e VALIDATE_GROOVY=true -e VALIDATE_BASH=true "$@" -v "$(projectRoot)"/:/tmp/lint github/super-linter
```

Alternatively we could add a directory argument to the script?  By default is uses `pwd` but in the library we could pass the path back to root?

Reply:
The addition of `projectRoot` is good, I've made the change.

Regarding where the script lives, I see a difference between the scripts stored in `./resources/**/scripts` and those in `./scripts`.
I see `./scripts` as a place users of repositories know to go to, by convention, to find useful scripts to help with the development of the repository, utilities, if you will. The scripts stored in `./resources/**/scripts`, I see as a part of what the repo does and not something that a developer would want to be running locally, at least on any kind of regular basis.
In this case the lines are blurred a bit because of the script in question is both a utility for developers and also something that happens to be used within the main function of the repo, however, I see the fact it is used within the repo function (i.e. the build process) as secondary as the linting could well have been included without the addition of a script.